src/tickable.ts

```diff
@@ -8,8 +8,52 @@ import { Vex } from './vex';
 import { Element } from './element';
 import { Flow } from './tables';
 import { Fraction } from './fraction';
+import { TickContext } from './tickcontext';
+import { ModifierContext } from './modifiercontext';
+import { Tuplet } from './tuplet';
+import { Voice } from './voice';
+import { Modifier } from './modifier';
+
+export interface LeftRight {
+  left: number;
+  right: number;
+}
+export interface Space {
```

Comment:
How about `Spacing`?

Reply:
I would leave it as Space. If you search in the code, you will find many references to `space` but none to `spacing`. Your choice anyway.

Reply:
Okay, it's fine.